src/tests/ftest/util/agent_utils_params.py

```diff
@@ -32,6 +33,29 @@ def _get_new(self):
         return DaosAgentTransportCredentials(self._log_dir)
 
 
+class DaosAgentTelemetryCredentials(TelemetryCredentials):
```

Comment:
Same comment as about the TelemetryCredentials elsewhere.

Reply:
Will update